src/Sut.WinForms.ScreenObjectsTest/ScreenObjectsTest.cs

```diff
@@ -106,6 +106,16 @@ public void Application()
         }
 
         [TestMethod]
+        public void FromProcess()
```

Comment:
I would like you to add the same test in WPF, since both WPF and WinForms are desktop application technologies, and thus should be supported by the `Screen` class.